deploy/chart/templates/14-operatorgroup.crd.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: operatorgroups.operators.coreos.com.operators.coreos.com
```

Comment:
Is this supposed to be "operatorgroups.operators.coreos.com"?

Reply:
Oh, yep. I think that's a kubebuilder bug as I don't think I messed that part up...